models/ed/inst/ED2IN.r2.2.0.github

```diff
@@ -599,11 +599,13 @@ $ED_NL
    !     They are used only when ISOILFLG is 2, both values are between 0. and 1., and     !
    ! their sum doesn't exceed 1.  In case ISOILFLG is 2 but the fractions do not meet the  !
    ! criteria, ED-2 uses NSLCON instead.                                                   !
+   ! NOTE: these defaults are arbitrary, but I don't think that matters since              !
+   ! SOIL_HYDRO_SCHEME=0                                                                   !
    !---------------------------------------------------------------------------------------!
-   NL%SLSOC   = myslsoc
-   NL%SLPH    = myslph
-   NL%SLCEC   = myslcec
-   NL%SLDBD   = mysldbd
+   NL%SLSOC   = 0.06
+   NL%SLPH    = 6
+   NL%SLCEC   = 0.5
+   NL%SLDBD   = 1330
```

Comment:
Not sure I understand why you switch out variables for hard-coded constants. Doesn't this eliminate the ability for PEcAn to set these variables?

Reply:
These are new variables that didn't exist in previous versions of ED2IN.  I assumed I would need knowledge of soil chemistry to connect these up to PEcAn in some way, but if that's not the case I can take a look at how other ED2IN variables are done and take a crack at this. Also, I'm pretty sure they aren't used unless another hard-coded variable is changed.

Reply:
So if the `mysl*` variables don't exist anywhere else in the code I'm OK with this PR, though I do think the soil.nc files do have most of these variables, if one wanted to set up the ability to specify these via code.

Reply:
I'll open an issue as a reminder to add this in the future